openwisp_radius/tests/mixins.py

```diff
@@ -65,6 +70,24 @@ def setUp(self):
 
 
 class BaseTestCase(DefaultOrgMixin, CreateRadiusObjectsMixin, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        if not os.path.isdir(settings.MEDIA_ROOT):
+            os.mkdir(settings.MEDIA_ROOT)
+
+    def tearDown(self):
+        for radbatch in RadiusBatch.objects.all():
+            # Delete associated pdf
+            if radbatch.pdf and os.path.isfile(radbatch.pdf.path):
+                os.remove(radbatch.pdf.path)
+            # Delete associated csv
+            if radbatch.csvfile and os.path.isfile(radbatch.csvfile.path):
+                os.remove(radbatch.csvfile.path)
```

Comment:
why not do this in the delete method of the model and have tearDown call radbatch.delete()?

Reply:
Great, Idea, I didn't have to do much to implement it! :smile: